ldk/javascript/packages/create-loop/index.js

```diff
@@ -0,0 +1,239 @@
+#!/usr/bin/env node
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs/promises');
+const path = require('path');
+const { execSync } = require('child_process');
+const prompts = require('prompts');
+const Sqrl = require('squirrelly');
+
+const projectOptions = {
+  name: '',
+  language: '',
+  aptitudes: [],
+};
+
+const installNodeModules = async () => {
+  try {
+    const { name: projectName } = projectOptions;
+
+    execSync('npm install', { cwd: `./${projectName}`, stdio: 'inherit' });
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+const createProject = async () => {
+  try {
+    const { name: projectName, aptitudes, language } = projectOptions;
+
+    // Render a given template file to a given target file path
+    const renderTemplate = async (template, filePath) => {
+      const fileContents = await Sqrl.renderFile(template, {
+        isTypeScript: language === 'typescript',
```

Comment:
since you do this check more than once, maybe create this variable as a const under line 28, the you can reference isTypeScript on lines 33 and 44

Reply:
Great catch - changed this!